### PR TITLE
added event.Topics

### PR DESCRIPTION
**Pull request by Jpiv1212**
### Please reference an issue, or fill out the below information

- Issue: #50

### CODE REVIEW

**Comment by jghibiki** [comment]
@dekarrin @Jpiv I dont think python's enum class is easily serializable this might create problems if/when network concepts are introduced. Just an observation.

**Comment by Jpiv1212** [comment]
> @dekarrin @Jpiv1212 I dont think python's enum class is easily serializable this might create problems if/when network concepts are introduced. Just an observation.

@jghibiki I looking into the documentation on enums beforehand, and from what I can tell there should be no problem with pickling these Enums. It is only instances born of a called Enum that runs into problems when pickling.

**Comment by Jpiv1212** [comment]
@dekarrin changes should good now, enum int values are now arbitrary

**Comment by dekarrin** [comment]
Rest assured, @jghibiki, we do not plan on using pickling for network transference, necessarily... But does this PR imply that, or push us in that direction? I'm not seeing how that would be an issue we can't easily work around.

File: python/msa/event.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from enum import Enum, auto
+
+
+class Topics(Enum):
+    EVENT_STACK_CLEARED = auto()
```
Comment (dekarrin) [inline python/msa/event.py]:
```suggestion
    STACK_CLEARED = (auto(), 0)
```

I was thinking more something like this for the constant declarations.

**Pull request merged**